cmake/Git.cmake

```diff
@@ -0,0 +1,31 @@
+#=============================================================================
+# Copyright 2011 Kitware, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is this tiny file really Apache 2? I thought it would be BSD like most of the CMake code in our projects. Guessing maybe it came from ITK or something?